test_common/helpers/setup.rb

```diff
@@ -310,12 +327,16 @@ def answer_all_questions # rubocop:disable Metrics/MethodLength
 
         employers_contract_claim_page.next
 
+        answer_upload_additional_information_question
+
+        additional_information_page.next
+
         answer_email_receipt_question
       end
 
-      # Stub Calls to API
+      # Stub Submission Calls to API
       def stub_et_api # rubocop:disable Metrics/MethodLength
-        stub_request(:post, "https://et-api-example.com/v2/respondents/build_response").
+        stub_request(:post, "http://api.et.127.0.0.1.nip.io:3100/api/v2/respondents/build_response").
```

Comment:
If et.127.0.0.1.nip.io:3100 comes from an environment variable somewhere then best to read that environment variable instead of hard coding in this value.  Then, if someone runs it with a different value in the env var you will not get test failures

Reply:
Use the ENV var fetch first in 24c3403

Reply:
Whoops - missed one. Fixed in 717f9d9